src/post.proj

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <Import Project="dir.props" />
+  <Import Project="..\dir.targets" />
+
+  <!-- provides a hook for executing targets after the product and tests have been built -->
+  <Target Name="Build" DependsOnTargets="$(PostBuildDependsOn)" />
+
+  <Import Project="$(ToolsDir)Build.Post.targets" Condition="Exists('$(ToolsDir)Build.Post.targets')" />
```

Comment:
There is generally two ways to hook targets together. One is the DependsOnTargets and the other is via AfterTargets or BeforeTargets. See https://msdn.microsoft.com/en-us/library/ee216359.aspx